File: spologp_v/lista2sala-C/questao5.c
#include<stdio.h>

int main()
{
    int A, B, C;

    printf("Digite o valor A:");
    scanf("%i", &A);
    printf("Digite o valor B:");
    scanf("%i", &B);
    printf("Digite o valor C:");
    scanf("%i", &C);

    if ((A >= B) && (B >= C)) {
        printf("A ordem crescente é: %i, %i, %i\n", C, B, A);
    } else if ((A >= C) && (C >= B)) {
        printf("A ordem crescente é: %i, %i, %i\n", B, C, A);
    } else if ((B >= A) && (A >= C)) {
        printf("A ordem crescente é: %i, %i, %i\n", C, A, B);
    } else if ((B >= C) && (C >= A)) {
        printf("A ordem crescente é: %i, %i, %i\n", A, C, B);
    } else if ((C >= A) && (A >= B)) {
        printf("A ordem crescente é: %i, %i, %i\n", B, A, C);
    } else {
        printf("A ordem crescente é: %i, %i, %i\n", A, B, C);
    }
    
    return 0;
}

File: spologp_v/lista2sala-python/questao5.py
A = int(input("Digite o valor A:"))
B = int(input("Digite o valor B:"))
C = int(input("Digite o valor C:"))

if (A>B>C) or (A>C>B):
    print("A ordem crescente é:", C, B, A) or ("A ordem crescente é:", B, C, A)
elif(B>A>C) or (B>C>A):
    print("A ordem crescente é:", C, A, B) or ("A ordem crescente é:", A, C, B)
elif(C>B>A) or (C>A>B):
    print("A ordem crescente é:", A, B, C) or ("A ordem crescente é:", B, A, C)
else:
    print("Esses valores não podem ser dispostos em ordem crescente.")